lib/jazzy/config.rb

```diff
@@ -220,6 +220,11 @@ def expand_path(path)
         SourceDeclaration::AccessControlLevel.from_human_string(acl)
       end
 
+    config_attr :lint,
+      command_line: '--[no-]lint',
+      description: "Generate linter output.",
+      default: false
```

Comment:
However it plays out with the options, I'd want to give users flexibility on two fronts:

- Scripts should be able to programmatically detect when lint doesn't pass in order to trigger some project-specific action (e.g. email team, fail CI, Slack hook).
- Teams should be able to decide whether it's OK to build docs even if lint doesn’t pass.

To that end, I’d evaluate any options scheme against three use cases:

1. “always build docs but send notification with line output iff lint fails”
2. “notify and do not build docs if lint fails”
3. “no lint; just build docs”